ml_peg/calcs/supramolecular/plf547/calc_plf547.py

```diff
@@ -0,0 +1,352 @@
+"""Calculate PLF547 benchmark. 10.1021/acs.jcim.9b01171."""
+
+from __future__ import annotations
+
+import logging
+from pathlib import Path
+
+import ase
+from ase import units
+from ase.io import write
+import mlipx
+from mlipx.abc import NodeWithCalculator
+import numpy as np
+from tqdm import tqdm
+import zntrack
+
+from ml_peg.calcs.utils.utils import chdir, download_s3_data
+from ml_peg.models.get_models import load_models
+from ml_peg.models.models import current_models
+
+MODELS = load_models(current_models)
+
+KCAL_TO_EV = units.kcal / units.mol
+EV_TO_KCAL = 1 / KCAL_TO_EV
+
+DATA_PATH = Path(__file__).parent / "data"
+
+OUT_PATH = Path(__file__).parent / "outputs"
+DATA_PATH = (
+    download_s3_data(
+        filename="PLF547.zip", key="inputs/supramolecular/PLF547/PLF547.zip"
+    )
+    # / "PLF547"
+)
+print(DATA_PATH)
+
+
+class PLF547Benchmark(zntrack.Node):
+    """Benchmarking PLF547 subset ionic hydrogen bonds benchmark dataset."""
+
+    model: NodeWithCalculator = zntrack.deps()
+    model_name: str = zntrack.params()
+
+    # ------------------------------------------------------------
+    # PDB processing functions
+    # ------------------------------------------------------------
+    @staticmethod
+    def extract_charge_and_selections(
+        pdb_path: Path,
+    ) -> tuple[float, float, float, str, str]:
+        """
+        Extract charge and selection information from PDB REMARK lines.
+
+        Parameters
+        ----------
+        pdb_path
+            Path to pdb files.
+
+        Returns
+        -------
+        tuple[int, int, int, list, list]: A tuple containing:
```

Comment:
```suggestion
        tuple[int, int, int, str, str]: A tuple containing:
```
From the code/return type hint?